main.tf

```diff
@@ -1,10 +1,18 @@
+provider "aws" {
+  region = "eu-west-1"
```

Comment:
We shouldn't set the region in the module - this prevents it from being used in any other region.